packages/postgres-database/src/simcore_postgres_database/migration/versions/6e91067932f2_adding_resource_tracker_container_table.py

```diff
@@ -0,0 +1,57 @@
+"""adding resource tracker container table
+
+Revision ID: 6e91067932f2
+Revises: 52cf00912ad9
+Create Date: 2023-06-21 14:12:40.292816+00:00
+
+"""
+import sqlalchemy as sa
+from alembic import op
+from sqlalchemy.dialects import postgresql
+
+# revision identifiers, used by Alembic.
+revision = "6e91067932f2"
+down_revision = "52cf00912ad9"
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.create_table(
+        "resource_tracker_container",
+        sa.Column("container_id", sa.String(), nullable=False),
+        sa.Column("image", sa.String(), nullable=False),
+        sa.Column("user_id", sa.BigInteger(), nullable=False),
+        sa.Column("product_name", sa.String(), nullable=False),
+        sa.Column(
+            "service_settings_reservation_nano_cpus", sa.BigInteger(), nullable=True
+        ),
+        sa.Column(
+            "service_settings_reservation_memory_bytes", sa.BigInteger(), nullable=True
+        ),
+        sa.Column(
```

Comment:
Why are only these 2 nullable?

Reply:
I got info from DevOps that sometimes when doing some manual intervention/debugging we might remove them. But we will see as we use it and change accordingly.